examples/integrations/cdi/jpa/src/main/java/io/helidon/examples/integrations/cdi/jpa/HelloWorldResource.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.examples.integrations.cdi.jpa;
+
+import java.net.URI;
+import java.util.Objects;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+import javax.persistence.EntityNotFoundException;
+import javax.persistence.PersistenceContext;
+import javax.persistence.PersistenceException; // for javadoc only
+import javax.transaction.Status;
+import javax.transaction.SystemException;
+import javax.transaction.Transaction;
+import javax.transaction.Transactional;
+import javax.transaction.Transactional.TxType;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+/**
+ * A JAX-RS root resource class that manipulates greetings in a
+ * database.
+ *
+ * @author <a href="mailto:laird.nelson@oracle.com">Laird Nelson</a>
+ *
+ * @see #get(String)
+ *
+ * @see #post(String, String)
+ */
+@Path("")
+// @RequestScoped // see https://github.com/oracle/helidon/issues/363
```

Comment:
This is not correct. As long as the resource is part of a JAX-RS Application that is marked as `ApplicationScoped` the resource should be `RequestScoped`.
As this is the case in your application, you do not need to do so. 
Also as you return a class from the Application, Jersey will create a new instance for each request, so this is a strange combination (e.g. RequestScoped from point of Jersey, ApplicationScoped from point of CDI)

Reply:
What do you feel is correct here for me to do?  (It is of course perfectly acceptable to mix scopes.)

Reply:
Use `@RequestScoped` in your resource class. There is no issue. 
The #363 is related _only_ to cases where there is no `@ApplicationScoped` Application class defined

Reply:
OK will do